setup.cfg

```diff
@@ -24,10 +24,10 @@ classifiers =
     Operating System :: OS Independent
     Programming Language :: Python :: 3
     Programming Language :: Python :: 3 :: Only
-    Programming Language :: Python :: 3.6
     Programming Language :: Python :: 3.7
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
+    Programming Language :: Python :: 3.10
```

Comment:
Should we add support for Python 3.10 in this PR then (and maybe associate this PR to https://github.com/spyder-ide/qtpy/issues/269)?

Reply:
I don't have time for that, but I doubt QtPy 2 will have trouble working with Python 3.10 because we're not using specific languages features that will prevent it.

Reply:
Like @dalthviz , I don't understand why we would formally declare support for Python 3.10 before actually testing it, and this is out of scope for this PR, given this is the declared purpose of issue #269 , which I'll take care of one way or another before the final QtPy 2.0 release.